app/waf/test/acceptance/adc.controller.acceptance.ts

```diff
@@ -427,18 +426,21 @@ describe('AdcController test', () => {
         .send(adc)
         .expect(200);
 
-      response = await client
-        .get(prefix + '/adcs/' + response.body.adc.id)
-        .set('X-Auth-Token', ExpectedData.userToken)
-        .set('tenant-id', ExpectedData.tenantId)
-        .expect(200);
+      let checkFunc = async () => {
+        response = await client
+          .get(prefix + '/adcs/' + response.body.adc.id)
+          .set('X-Auth-Token', ExpectedData.userToken)
+          .set('tenant-id', ExpectedData.tenantId)
+          .expect(200);
 
-      expect(response.body.adc.status).to.equal(AdcState.TRUSTERR);
+        return response.body.adc.status === 'TRUSTERROR';
+      };
+      await checkAndWait(checkFunc, 50, [], 5).then(() => {
+        expect(true).eql(true);
```

Comment:
Do not recommend to write ut in this way. Your error root cause will be invisible.